content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -86,11 +86,8 @@ In the above example, this version is `kubelet.config.k8s.io/v1beta1`.
 
 ## Drop-in directory for kubelet configuration files {#kubelet-conf-d}
 
-As of Kubernetes v1.28.0, the kubelet has been extended to support a drop-in configuration directory. The location of it can be specified with
-`--config-dir` flag, and it defaults to `""`, or disabled, by default.
+In Kubernetes v1.30.0, the drop-in configuration directory feature has reached Beta status. This allows the kubelet to support a drop-in configuration directory, which can be specified using the `--config-dir` flag. By default, it is set to `""`, indicating that the feature is disabled by default.
```

Comment:
I think this should be 1.29.0 and update with the new correct behavior. We can update to beta once we actually update the code to beta

Reply:
Fixed